<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SizeFilter</class>
 <widget class="QDialog" name="SizeFilter">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>297</width>
    <height>105</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Size Filter</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QCheckBox" name="morecheckBox">
     <property name="text">
      <string>Show Items with size &gt;</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QSpinBox" name="morespinBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="maximum">
      <number>1000000000</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="lesscheckBox">
     <property name="text">
      <string>Show Items with size &lt; </string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="lessspinBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="maximum">
      <number>1000000000</number>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="pushButton">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Apply</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
